templates/email/order-confirmation.html.twig

```diff
@@ -0,0 +1,23 @@
font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:15px;line-height:1;text-align:center;color:#FFFFFF;"><strong>Total Price</strong></div></td></tr><tr><td align="center" style="font-size:0px;padding:10px 25px;padding-top:10px;padding-right:25px;padding-bottom:20px;padding-left:25px;word-break:break-word;"><div style="font-family:Helvetica;font-size:13px;line-height:1;text-align:center;color:#FFFFFF;">${{ cart.total }}</div></td></tr></tbody></table></div><!--[if mso | IE]></td></tr></table><![endif]--></td></tr></tbody></table></div><!--[if mso | IE]></td></tr></table><table align="center" border="0" cellpadding="0" cellspacing="0" class="" style="width:600px;" width="600" bgcolor="#356CC7" ><tr><td style="line-height:0px;font-size:0px;mso-line-height-rule:exactly;"><![endif]--><div style="background:#356CC7;background-color:#356CC7;margin:0px auto;max-width:600px;"><table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation" style="background:#356CC7;background-color:#356CC7;width:100%;"><tbody><tr><td style="direction:ltr;font-size:0px;padding:20px 0;padding-bottom:40px;padding-top:40px;text-align:center;"><!--[if mso | IE]><table role="presentation" border="0" cellpadding="0" cellspacing="0"><tr><td class="" style="vertical-align:top;width:600px;" ><![endif]--><div class="mj-column-per-100 mj-outlook-group-fix" style="font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;"><table border="0" cellpadding="0" cellspacing="0" role="presentation" style="vertical-align:top;" width="100%"><tbody><tr><td align="left" style="font-size:0px;padding:10px 25px;word-break:break-word;"><table cellpadding="0" cellspacing="0" width="100%" border="0" style="color:#FFFFFF;font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:13px;line-height:22px;table-layout:auto;width:100%;border:none;"><tr style="border-bottom:1px solid #ecedee;text-align:center;padding:15px 0;"><th style="padding: 0 15px 0 0;">Name</th><th style="padding: 0 15px;">Image</th><th style="padding: 0 0 0 15px;">Price</th><th style="padding: 0 0 0 15px;">Quantity</th><th style="padding: 0 0 0 15px;">Subtotal</th></tr>{% for orderProduct in cart.orderProducts %}<tr style="border-bottom:1px solid #ecedee;text-align:center;padding:15px 0;"><td style="padding: 0 15px 0 0;">{{ orderProduct.product.name }}</td><td style="padding: 0 15px;"><img src="{{absolute_url(asset(product_images_directory ~ orderProduct.product.imageLink))}}" alt="Product Image" width="100px" height="100px" border="none"></td><td style="padding: 0 0 0 15px;">${{ orderProduct.product.price }}</td><td style="padding: 0 0 0 15px;">{{ orderProduct.quantity }}</td><td style="padding: 0 0 0 15px;">${{ orderProduct.product.price * orderProduct.quantity }}</td></tr>{% endfor %}</table></td></tr></tbody></table></div><!--[if mso | IE]></td></tr></table><![endif]--></td></tr></tbody></table></div><!--[if mso | IE]></td></tr></table><![endif]--></div></body></html>
```

Comment:
??
please add indentation. this part is not reviewable.

Reply:
Actually this is auto generated by mjml and I'm using the minified version for optimization. To review you can check order-confirmation.mjml

Reply:
1. what is path of `order-confirmation.mjml`? I couldn't find it.
2. While Symfony takes care of minification of assets, what's the need of putting minified code? It results in unreviewable code, I suggest you to include the original version of code instead of minified.